### ISSUE TITLE
fab uninstall postgres cleanup errors when run from /opt/goldstone

### ISSUE THREAD

**Issue by heidecke** [Solinea/goldstone-server#33]
After a failed install, ran `fab uninstall` from /opt/goldstone and received the following error.

```

Dependencies Resolved

============================================================================================================================================
 Package                Arch         Version                                   Repository                                              Size
============================================================================================================================================
Removing:
 goldstone-server       x86_64       1431957243:0.5.0-0.5.0.el7.centos         @/goldstone-server-0.5.0-0.5.0.el7.centos.x86_64       3.6 M

Transaction Summary
============================================================================================================================================
Remove  1 Package

Installed size: 3.6 M
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Erasing    : 1431957243:goldstone-server-0.5.0-0.5.0.el7.centos.x86_64                                                                1/1
  Verifying  : 1431957243:goldstone-server-0.5.0-0.5.0.el7.centos.x86_64                                                                1/1

Removed:
  goldstone-server.x86_64 1431957243:0.5.0-0.5.0.el7.centos

Complete!
[localhost] local: su - postgres -c 'dropdb goldstone'
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
[localhost] local: su - postgres -c 'dropuser goldstone'
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
[localhost] local: systemctl stop postgresql.service
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
[localhost] local: systemctl disable postgresql.service
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
rm '/etc/systemd/system/multi-user.target.wants/postgresql.service'

Done.
```

**Comment by stugots** [comment]
Luke, if you can confirm that this is solved, please close this ticket.

**Comment by heidecke** [comment]
Resolved in 0.5.1.
